plugins/lookup/supported_roles.py

```diff
@@ -38,6 +38,7 @@
         'manage_dbpatches',
         'manage_efm',
         'setup_patroni',
+        'setup_pgd',
```

Comment:
I think the addition of the `setup_pgd` role should not be by default and should mirror the BDR implementation, this can assist with determining when to run the `init_dbserver` role instead of using a flag. Regardless, this file should be cleaned up of BDR items.